app/mailers/admin_mailer.rb

```diff
@@ -0,0 +1,27 @@
+class AdminMailer < ApplicationMailer
+  def notify_below_rate(alert)
+    repository = alert.repository
+    department = alert.department
+
+    @alert = alert
+    @metric_name = alert.metric_name
+    @entity_is_repo = repository.present?
+
+    return unless @entity_is_repo || department.present?
+
+    from = ENV['SENDGRID_USERNAME']
+    @entity_name = @entity_is_repo ? repository.name : department.name
+
+    mail(from: from,
+         to: notify_below_rate_emails,
+         subject: I18n.t('mailer.notify_below_rate.subject'))
+  end
+
+  def notify_below_rate_emails
+    @alert.emails.split(',').collect(&:strip) || self.class.admin_emails.collect(&:strip)
+  end
+
+  def self.admin_emails
+    AdminUser.pluck(:email)
```

Comment:
wouldn't this be potentially leaking information about the admin users that exist in the system?